src/core/DataFile.cpp

```diff
@@ -1386,6 +1387,84 @@ void DataFile::upgrade_noHiddenClipNames()
 }
 
 
+/** \brief Note range has been extended to match MIDI specification
+ *
+ * The non-standard note range previously affected all MIDI-based instruments
+ * except OpulenZ, and made them sound an octave lower than they should (#1857).
+ */
+void DataFile::upgrade_extendedNoteRange()
+{
+	auto affected = [](QDomElement instrument)
```

Comment:
```suggestion
	auto affected = [](const QDomElement& instrument)
```
Or alternatively, you can pass the name instead of the full element.

Reply:
Oh, did not occur to me to make it const, thanks.